Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -120,6 +123,10 @@ public class Environment: Environmenting {
         }
     }
 
+    public var testsCacheDirectory: AbsolutePath {
```

Comment:
we store the hashes of the targets that were part of a successful test run. I will think about better naming, I can't think of any right now

Reply:
I don't think changing the name makes sense -> or change all the other directories located in cache which also have `Cache` in their name. But honestly, I think it's better as is because in code is used as `Environment.shared.testsCacheDirectory`. In that place you don't have to know that the `testsDirectory` is actually in `Cache` directory, so this way is imho more explicit and clearer.

Reply:
Fair enough